pkg/zero/importutil/namegen_test.go

```diff
@@ -392,3 +394,13 @@ func TestGenerateRouteNames(t *testing.T) {
 		})
 	}
 }
+
+func TestGenerateRouteNamesForPolicy(t *testing.T) {
```

Comment:
Could add this to the other test instead, for example 

```diff
	for _, tc := range cases {
		t.Run(tc.name, func(t *testing.T) {
			assert.Equal(t, tc.expected, importutil.GenerateRouteNames(tc.input))
+			policies := make([]*config.Policy, len(tc.input))
+			for i := range tc.input {
+				policies[i], _ = config.NewPolicyFromProto(tc.input[i])
+			}
+			assert.Equal(t, tc.expected, importutil.GenerateRouteNames(policies))
```

Reply:
Good idea!